### PR TITLE
"[script.module.pyxbmct] 1.2.0"

### PR DESCRIPTION
**Pull request by romanvm**
### Description
This update adds Estuary-based skin for PyXBMCt windows/controls to be consistent with the new Kodi 17 look and also fixes some bugs.
### Checklist:
- [ x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [ x] Each add-on submission should be a sinlge commit with using the following style: [sciprt.foo.bar] v1.0.0

### CODE REVIEW

File: script.module.pyxbmct/lib/pyxbmct/addonskin.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        Get the base directory for image files
+
+        :rtype: str
+        """
+        return
+
+    @abstractproperty
+    def x_margin(self):
+        """
+        Get horisontal adjustment for the header background
```
Comment (phil65) [inline script.module.pyxbmct/lib/pyxbmct/addonskin.py]: typo, "horizontal"

File: script.module.pyxbmct/lib/pyxbmct/addonskin.py [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+        if the main background has transparent edges.
+
+        :rtype: int
+        """
+        return
+
+    @abstractproperty
+    def title_bar_x_shift(self):
+        """
+        Get horisontal adjustment for title bar texture
```
Comment (phil65) [inline script.module.pyxbmct/lib/pyxbmct/addonskin.py]: same

File: script.module.pyxbmct/lib/pyxbmct/addonskin.py [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        Get the height of the top-right close button
+
+        :rtype: int
+        """
+        return
+
+    @abstractproperty
+    def close_btn_x_offset(self):
+        """
+        Get close button horisontal adjustment
```
Comment (phil65) [inline script.module.pyxbmct/lib/pyxbmct/addonskin.py]: same

**Comment by phil65** [comment]
Just some typos in docstrings :D
Have you thaught about submitting this new version to Krypton repository? That way users of Jarvis and lower will get your confluence version, and krypton users get this new one with estuary. Perhaps you might be able to drop backwards compatibility that way?

**Comment by romanvm** [comment]
Typos are fixed.

As for other stuff, I always wanted to have several switchable skins but since I'm no designer I was stuck with only one until now. And I don't like the idea of introducing breaking changes or maintaining several versions, unless absolutely necessary. This is not to mention that this is a bugfix release too that fixes a serious long-standing bug. It skipped my attention until now, because it seems that `Blank*` classes are used almost never.

BTW, `CONTRIBUTING.md` has a couple of errors regarding git usage. I guess I need to make a PR for that too. And maybe share my continuous integration config that does all this stuff automatically, maybe in the Kodi Wiki.

**Comment by phil65** [comment]
Ok, cool. Just wanted to make sure that you know about that option.

**Comment by MartijnKaijser** [comment]
@romanvm in case you ever update

> PROBLEM: Not found LICENSE\.txt|LICENSE\.md|LICENSE in folder /home/travis/build/xbmc/repo-scripts/script.module.pyxbmct

**Comment by romanvm** [comment]
Regarding license, it's a false positive. The library does include GPLv3 license. Normally, I don't use ALL UPPERCASE file names unless absolutely necessary. Noted about an icon.

**Comment by MartijnKaijser** [comment]
@razze could you adjust the script for this?

**Pull request merged**